content/appendices.md

```diff
@@ -745,7 +745,7 @@ Specifically, the following mappings are used:
 * `r` for room aliases.
 * `u` for users.
 * `roomid` for room IDs (note the distinction from room aliases).
-* `e` for events, when after a room reference (`r` or `roomid`).
+* `e` for events, when after a room reference (`roomid`).
```

Comment:
The MSC specifically says, of event URIs using room aliases "Clients should ... continue to accept them"... so it seems a bit premature to rip them out here. (Yes, there's an INFO box, but nobody's going to read that.)

Perhaps we could move the INFO boxes right to the bottom of their respective sections, and just say:

```suggestion
* `e` for events, when after a room ID (`roomid`). Use of `e` after a room alias (`r`) is deprecated.
```